content/en/docs/configuration/acme/dns01/route53.md

```diff
@@ -225,6 +225,35 @@ metadata:
     eks.amazonaws.com/role-arn: arn:aws:iam::XXXXXXXXXXX:role/cert-manager
 ```
 
+Make sure there is no `role: arn:aws:iam::YYYYYYYYYYYY:role/dns-manager` field on the issuer definition, otherwise you might end up with `sts:assumerole` permission error.  
+
+Example of `issuer` or `clusterissuer` EKS OIDC definition:
+
+```yaml
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-dvl
+spec:
+  acme:
+    email: user@email.com
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: tls-key
+    solvers:
+    - selector:
+        dnsZones:
+          - "foobar.cloud"
+          - "*.foobar.cloud"
+      dns01:
+        route53:
+          region: us-east-1
+          hostedZoneID: ABCD123
+
+```
+
+To check for errors, create a `certificate` resource to kickstart the process and monitor  `cert-manager` pod logs.
```

Comment:
It is not advice to use the cert-manager logs for investigating issues, cert-manager reports all events back to Kubernetes which you can see using `kubectl describe`

Reply:
> It is not advice to use the cert-manager logs for investigating issues, cert-manager reports all events back to Kubernetes which you can see using `kubectl describe`

Hi! I see, I did not recall checking the output of `describe` tbh. Unfortunately I won't be able to test the scenario anytime soon. What about closing the PR and re-openning it when we can test the error log appears when describing the resource? I might be able to do so in a few weeks.

Reply:
Is good! feel free to close it!